README.md

```diff
@@ -1,76 +1,81 @@
 
-> NOTE: On Windows after installing the graphViz msi/zip, you will most likely need to add your PATH env variable `(Ex. c:\Program Files(x86)\GraphViz2.38\bin)` in order to use dot from the command line.
+> NOTE: Please make sure these requirements are in your PATH environment variable.
+
+## Features
+
+The features in this extension support execution in integrated terminal mode or remotely using Azure Cloud Shell. Some features only run locally at this time and will require some local dependencies.
+
+This extension supports the following features:
 
-## Supported Environments
+- Terraform commands: init, plan, apply, validate, refresh and destroy.
+- Visualize the terraform module.
 
-- [Microsoft Azure](https://azure.microsoft.com)
+## Commands
+
+Open the Command Palette (`Command`+`Shift`+`P` on macOS and `Ctrl`+`Shift`+`P` on Windows/Linux) and type in one of the following commands:
+
+<table>
+  <thead>
+  <tr>
+    <th>Command</th>
+    <th>Description</th>
+  </tr>
+  </thead>
+  <tbody>
+  <tr>
+    <td width="35%">
+      Basic commands:<br>
+      <ul>
+        <li>Azure Terraform: init</li>
+        <li>Azure Terraform: plan</li>
+        <li>Azure Terraform: apply</li>
+        <li>Azure Terraform: validate</li>
+        <li>Azure Terraform: refresh</li>
+        <li>Azure Terraform: destroy</li>
+      </ul>
+    </td>
+    <td>
+      Execute terraform command against the current project workspace.
+      If run with terminal set to Cloud Shell, will run the command in Cloud Shell.
+    </td>
+  </tr>
+  <tr>
+    <td>Azure Terraform: visualize</td>
+    <td>Create a visual representation of the components of the module and save it in <code>graph.png</code>.</td>
+  </tr>
+  <tr>
+    <td>Azure Terraform: push</td>
+    <td>Push workspace files that meet the filter <code>azureTerraform.files</code> setting in your configuration to Cloud Shell.</td>
+  </tr>
+  </tbody>
+</table>
 
 ## Extension Settings
 
 - `azureTerraform.terminal` - Specifies terminal used to run Terraform commands. Valid settings are `cloudshell` or `integrated`
 - `azureTerraform.files` - Indicates the files that should be synchronized to Azure CloudShell using the glob pattern string, for example `**/*.{tf,txt,yml,tfvars,rb}`
 
-## Known Issues
-
-- Windows support for dot has some unhandled exception cases.  We are working to improve this area.
-- We do not support private registry. The test container has to be on docker hub at this time.
-
 ## Release Notes
 
 Refer to [CHANGELOG](CHANGELOG.md)
+
+## Telemetry
+VS Code collects usage data and sends it to Microsoft to help improve our products and services. Read our [privacy statement](https://go.microsoft.com/fwlink/?LinkID=528096&clcid=0x409) to learn more. If you don’t wish to send usage data to Microsoft, you can set the `telemetry.enableTelemetry` setting to `false`. Learn more in our [FAQ](https://code.visualstudio.com/docs/supporting/faq#_how-to-disable-telemetry-reporting).
```

Comment:
If you would like to opt out of sending telemetry data to Microsoft, update the `telemetry.enableTelemetry` setting to `false` in the **File** -> **Preferences** -> **Settings**. To learn more see our [FAQ](https://code.visualstudio.com/docs/supporting/faq#_how-to-disable-telemetry-reporting).

Reply:
Thanks, updated.